src/js/components/TextInput/TextInput.js

```diff
@@ -361,6 +361,11 @@ const TextInput = forwardRef(
           {...dropProps}
         >
           <ContainerBox
+            aria-label={
```

Comment:
@jcfilben @taysea 
I can go back and add this into our default.json message and allow for internationalization but wanted to have you guys check first if the voice over made sense while going through the default suggestion story